src/main/java/com/mateacademy/springmvc/interceptor/UserInterceptor.java

```diff
@@ -0,0 +1,30 @@
+package com.mateacademy.springmvc.interceptor;
+
+import lombok.extern.log4j.Log4j;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@Component
+@Log4j
+public class UserInterceptor extends HandlerInterceptorAdapter {
+    @Override
+    public boolean preHandle(HttpServletRequest requestServlet,
+                             HttpServletResponse responseServlet, Object handler) throws Exception {
+        log.info("-----Interceptor preHandle called");
+        return true;
+    }
+
+    @Override
+    public void postHandle(HttpServletRequest request, HttpServletResponse response, Object handler, ModelAndView modelAndView) throws Exception {
```

Comment:
Line with the code is too long. Please break it into two lines like in lines 15 and 16.